docs/NATIVE_SPEND.md

```diff
@@ -1,149 +1,166 @@
+
+*To add a custom Spend offer to the Kin Marketplace:*
+
+1. Create a ```NativeSpendOffer``` object as in the example below.
+
 ```java
-    NativeSpendOffer nativeOffer =
-        new NativeSpendOffer("The offerID") // OfferId should be uniqe
-            .title("Offer Title")
-            .description("Offer Description")
-            .amount(1000)
-            .image("A URL to offer image");
+NativeSpendOffer nativeOffer =
+        new NativeSpendOffer("The offerID") // OfferId must be a UUID
+            .title("Offer Title") // Title to display with offer
+            .description("Offer Description") // Desc. to display with offer
+            .amount(1000) // Purchase amount in Kin
+            .image("Image URL"); // Image to display with offer
 ```
-3. Add the offer to Kin Marketplace, notice each offer you add will be added at index 0.
+2.	Create a ```NativeOfferObserver``` object to be notified when the user clicks on your offer in the Kin Marketplace.
+
+>**NOTE:** You can remove the Observer by calling ```Kin.removeNativeOfferClickedObserver(…)```.
+
 ```java
-    try {
-        if (Kin.addNativeOffer(nativeSpendOffer)) {
-            // Native offer added
-        } else {
-            // Native offer already in Kin Marketplace
+private void addNativeOfferClickedObserver() {
+try {
+
+Kin.addNativeOfferClickedObserver(getNativeOfferClickedObserver());
+        } catch (TaskFailedException e) {
+            showToast("Could not add native offer callback");
         }
-    } catch (ClientException error) {
-        ...
     }
+
+    private Observer<NativeOfferClickEvent> getNativeOfferClickedObserver() {
+        if (nativeSpendOfferClickedObserver == null) {
+            nativeSpendOfferClickedObserver = new Observer<NativeOfferClickEvent>() {
+                @Override
+                public void onChanged(NativeOfferClickEvent nativeOfferClickEvent) {
+                    NativeSpendOffer nativeSpendOffer = (NativeSpendOffer) nativeOfferClickEvent.getNativeOffer();
+                    if(nativeOfferClickEvent.isDismissOnTap()){
+                        new AlertDialog.Builder(MainActivity.this)
+                            .setTitle("Native Offer (" + nativeSpendOffer.getTitle() +")")
+                            .setMessage("You tapped a native offer and the observer was notified.")
+                            .show();
+                    } else {
+                        Intent nativeOfferIntent = NativeOfferActivity.createIntent(MainActivity.this, nativeSpendOffer.getTitle());
+                        startActivity(nativeOfferIntent);
+                    }
+                }
+            };
+        }
+        return nativeSpendOfferClickedObserver;
+}
+```
+
+3.	```Call Kin.addNativeOffer(nativeSpendOffer, dismissOnTap)```.
```

Comment:
One tick, also the `Call` part should probably be outside of the tick altogether:
```
Call `Kin.addNativeOffer(nativeSpendOffer, dismissOnTap)`
```